server/loadConfigs.js

```diff
@@ -58,6 +58,18 @@ function loadClientConfigs(_config) {
     name => name[0] !== '.' && /.conf$/.test(name)
   );
 
+  // cancel startup process if there are no clientConfigs
+  if (!files.length) {
+    logger.error('No client configs found.');
+    logger.error(
+      'Please add at least one config called "default" to your clients directory and try again.'
+    );
+    logger.error(
+      'Visit the documentation for more information: https://bpanel.org/docs/configuration.html'
```

Comment:
Is there any chance that this URL will change?

Reply:
Possibly, but I think we just have to deal with it when it happens. I'm not sure how other libraries do it (React has great error logs with links to possible solutions for example), except that we should probably add redirects on the documentation website side.